kano_updater/scenarios.py

```diff
@@ -797,7 +837,7 @@ def _bootconfig_set_value_helper(self, setting, value):
         except ImportError:
             logger.error("end_config_transaciton not present - update to kano-settings failed?")
 
-    def beta_370_to_beta_380(self):
+    def beta_370_to_beta_380(self, dummy_progress):
```

Comment:
I like that the scenarios are now able to report on progress back to the GUI. This scenario currently is a problem because it installs 11 apps without any progress indication. Could we create a ticket to fix this for later?